grin/test/AbstractInterpretation/ExtendedSyntax/IRSpec.hs

```diff
@@ -42,42 +41,102 @@ randomizeInstructions is0 = do
     rest -> pure rest
 
 testProgram :: Exp
-testProgram = convertToNew [Old.prog|
-    grinMain = t1 <- store (CInt 1)
-               t2 <- store (CInt 10000)
-               t3 <- store (Fupto t1 t2)
-               t4 <- store (Fsum t3)
-               (CInt r') <- eval t4
-               _prim_int_print r'
+testProgram = [prog|
+  grinMain =
+    v.0 <- do
+      y.0 <- pure 1
+      pure (CInt y.0)
```

Comment:
why `do`? why isn't it bind normalised?

Reply:
I forgot a bind normalisation in the conversion function (old AST -> new AST).

Reply:
Done.